packages/web-components/src/components/back-to-top/back-to-top.ts

```diff
@@ -90,17 +102,28 @@ class DDSBackToTop extends HostListenerMixin(StableSelectorMixin(LitElement)) {
     }
   }
 
-  private _observeResizeBody = (entries) => {
+  /**
+   * ResizeObserver callback function.
+   *
+   * We only observe the body element, therefore entries will typically only
+   * have a single item corresponding to changes in the size of the body
+   * element.
+   */
+  private _observeResizeBody = (entries: ResizeObserverEntry[]) => {
```

Comment:
I think we probably ought to rename this function becuase `observeResizeBody` and `observerResizeBody` is just _begging_ for typos/mis-understandings.

I'd suggest `handleBodyResize` here.

Reply:
Good thought. Updated.